GPT/gpt.py

```diff
@@ -192,20 +195,32 @@ def gpt_reformat_last(how_to_reformat: str):
             notify("No text to reformat")
             raise Exception("No text to reformat")
 
+    def gpt_paste_with_modifier(text: str, modifier: str = ""):
```

Comment:
I think I would just call this `gpt_insert`. And I would be tempted to just have `modifier` be a boolean `is_snippet` for now as it's unclear we'll actually need more modes, and the name `modifier` is quite vague